packages/app/package.json

```diff
@@ -18,42 +18,44 @@
     "lint:js": "eslint .",
     "lint": "npm run lint:hbs && npm run lint:js",
     "start": "ember serve",
+    "pretest": " ember backstop-remote& ",
```

Comment:
Can we check if the backstop environment var? Also, can we have a script for the docker command?